src/compas_ghpython/components_cpython/Compas_RpcCall/code.py

```diff
@@ -0,0 +1,32 @@
+"""
+Remote Procedure Call: to invoke Python functions outside of Rhino, in the context of the CPython interpreter.
+"""
+
+# r: compas==2.8.1
+from ghpythonlib.componentbase import executingcomponent as component
+
+from compas.rpc import Proxy
+
+
+class CompasRpcCall(component):
```

Comment:
maybe consider `CompasRemoteCall`... procedure seems not relevant to mention and `c` and `Call` are the same.